### ISSUE TITLE
[InputNew]: Minor bug

### ISSUE THREAD

**Issue by AbhinavMV** [web3ui/web3uikit#1032]
only happens when id is not given to the input
when multiple inputs are present on same form , clicking on bottom inputs label focuses on the first input

Reason: by default id is `undefined` and so all inputs have same id, and since label has `htmlFor=id` , the first input is focused

Solution: give a unique id to each input as default or make id field as mandatory (making field mandatory will be breaking change so discuss before adding it) 

https://user-images.githubusercontent.com/35369843/218090441-f1b802a0-a966-4d08-918c-235438aad112.mov

**Comment by BillyG83** [comment]
mandatory sounds like a good idea to me. TS complier will find it and JS users wont care anyway

**Comment by AbhinavMV** [comment]
yeah sounds good to me but it means whenever this is fixed we have to fix everything in admin too

**Comment by BillyG83** [comment]
@AbhinavMV do you guys have a `build` command. I think it would pick them all up and make a list for you guys. Yeah we should be more careful i think everyone got into a habit of making almost all props optional 😅
